ode

```diff
@@ -0,0 +1,4 @@
+# Netscape HTTP Cookie File
```

Comment:
Is there a reason to commit this file?

Reply:
@paskal Removed the file, added by a mistake